### PR TITLE
OLMIS-6315: added requisition status filter

### PR DESCRIPTION
**Pull request by mkwiatkowskisoldevelo**

### CODE REVIEW

**Comment by mkwiatkowskisoldevelo** [comment]
> Changes look good for me but is it okay that user can select only one requisition status? From what I see API accepts a list of statuses. Is it possible that a user will try to find requisitions by several statuses?

The problem is that we don't have multiple selection input on the UI for now.

**Comment by lukaslew** [comment]
@mkwiatkowskisoldevelo Make sense

File: src/requisition-search/requisition-search.controller.js [hunk 1]
```diff
@@ -239,10 +268,26 @@
...
+         * @methodOf requisition-search.controller:RequisitionViewController
+         * @name getStatusDisplay
+         *
+         * @description
+         * Evaluates display name for given status.
+         * 
+         * @param   {String} status given status
+         * @returns {String}        translated status name
+         */
+        function getStatusDisplay(status) {
```
Comment (ngraczewski) [inline src/requisition-search/requisition-search.controller.js]: Can't we use the filter directly in the html?

Comment (mkwiatkowskisoldevelo) [reply src/requisition-search/requisition-search.controller.js]: Done

**Pull request merged**